src/style.css

```diff
@@ -1,3 +1,108 @@
-body {
+html {
+  background-color: #e5e5e5;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-style: normal;
+
+  --secondary: #616070;
+  --main-accent: #060b35;
+  --white: #fff;
+  --blue-accent: #00acdc;
+}
+
+body,
+div,
+a,
+h1,
+h2,
+img,
+p {
   margin: 0;
+  padding: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 16px 16px;
+}
+
+.card__imac {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__imac-title {
+  color: var(--main-accent);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-bottom: 4px;
+}
+
+.card__imac-code {
+  color: var(--secondary);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  align-self: flex-start;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  color: var(--secondary);
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  align-self: flex-start;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__title-price {
+  color: var(--main-accent);
+  text-align: right;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 18px;
+  margin: 0 86px 16px;
+}
+
+.card__button {
+  flex-shrink: 0;
+  width: 166px;
+  display: block;
+  color: var(--white);
+  text-align: center;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 40px;
+  text-transform: uppercase;
+  border-radius: 5px;
+  background: var(--blue-accent);
+  text-decoration: none;
+}
+
+.card__button:hover {
+  flex-shrink: 0;
+  width: 166px;
+  display: block;
+  color: var(--blue-accent);
+  text-align: center;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 38px;
+  text-transform: uppercase;
+  text-decoration: none;
+  border-radius: 5px;
+  background: #fff;
+  border: 1px solid var(--blue-accent);
```

Comment:
There is no need to duplicate all properties. Set only the properties that change on hover